docker/docker-compose.yml

```diff
@@ -0,0 +1,65 @@
+version: "3"
+name: pagopa-interop
+services:
+  catalog-event-store:
+    volumes:
+      - ./event-store-init.sql:/docker-entrypoint-initdb.d/01-init.sql
+    image: postgres:14
+    ports:
+      - 6001:5432
+    environment:
+      POSTGRES_DB: root
+      POSTGRES_USER: root
+      POSTGRES_PASSWORD: root
+    command: -c 'max_connections=512' -c 'wal_level=logical'
+
+  zookeeper:
+    image: quay.io/debezium/zookeeper:2.3
+    ports:
+      - 2181:2181
+      - 2888:2888
+      - 3888:3888
+  kafka:
+    image: quay.io/debezium/kafka:2.3
+    ports:
+      - 9092:9092
+      - 29092:29092
+    links:
+      - zookeeper
+    environment:
+      ZOOKEEPER_CONNECT: zookeeper:2181
+      KAFKA_LISTENERS: LISTENER_DOCKER://kafka:29092,LISTENER_HOST://kafka:9092
+      KAFKA_ADVERTISED_LISTENERS: LISTENER_DOCKER://kafka:29092,LISTENER_HOST://localhost:9092
+      KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: LISTENER_DOCKER:PLAINTEXT,LISTENER_HOST:PLAINTEXT
```

Comment:
we might be able to simplify this configuration now everything is in the same Docker network, but I'll have a look at this in a subsequent PR